css/style.css

```diff
@@ -26,11 +24,150 @@ video {
 }
 
 .container {
-  margin: 0 auto;
+	position: relative;
+	margin: 0 auto;
   max-width: 960px;
   width: 100%;
+  height: 100%;
+}
+
+/* =========== MAIN-NAV MOBILE ============ */
+.main-nav {
+	position: fixed;
+	top: 0;
+	z-index: 2;
+	width: 100%;
+	height: 75px;
+	overflow-x: hidden;
+	transition: all 0.5s;
+}
+
+.main-nav--colored {
+	background-color: rgba(0, 60, 143, 0.8);
+}
+
+.main-nav__logo {
+	position: absolute;
+	top: 11px;
+	left: 20px;
+}
+
+.main-nav__logo__image {
+	width: 89px;
+}
+
+.main-nav__burguer {
+	position: absolute;
+	top: 20px;
+	right: 20px;
+	cursor: pointer;
+}
+
+.main-nav__burguer-bar {
+	width: 40px;
+	height: 7px;
+	background-color: white;
+	transition: 0.5s;
+}
+
+.burguer__bar-1,
+.burguer__bar-2 {
+	margin-bottom: 7px;
+}
+
+.burguer-animation .burguer__bar-1 {
+	transform: rotate(-45deg) translate(0px, 14px);
+	transform-origin: center center;
+	margin: 0;
+}
+
+.burguer-animation .burguer__bar-2 {
+	opacity: 0;
+}
+
+.burguer-animation .burguer__bar-3 {
+	transform: rotate(45deg) translate(0px, -14px);
+	transform-origin: center center;
+}
+
+.main-nav__menu-items {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	position: absolute;
+	top: 0;
+	z-index: -1;
+	width: 100%;
+	height: 0;
+	margin: 0;
+	padding: 0;
+	opacity: 0;
+	list-style-type: none;
+	overflow: hidden;
+}
+
+.main-nav__menu-link {
+	color: white;
+	font-family: roboto, sans-serif;
```

Comment:
Since this font is set for the whole body at the beginning of this file, this line is not needed.

Reply:
Done, thank you.